pkg/collector/process.go

```diff
@@ -156,15 +156,15 @@ func (cp *collectingProcess) decodeTemplateSet(templateBuffer *bytes.Buffer, obs
 			}
 		} else {
 			/*
-					Encoding format for Enterprise-Specific Information Elements:
-				     0                   1                   2                   3
-					 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-					+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-					|1| Information Element id. = 15 | Field Length = 4 |
-					+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+++
-					| Enterprise number |
-					+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-					(Reference: https://tools.ietf.org/html/rfc7011#appendix-A.2.2)
+				Encoding format for Enterprise-Specific Information Elements:
+				 0                   1                   2                   3
+				 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+				+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+				|1| Information Element id. = 15 | Field Length = 4 |
```

Comment:
Sorry for nit picking. The number of bits/bytes used is still unclear. Instead of values, we can add bits in parentheses 
1 (1bit) | Information Element Id(15bits) Field Length (16bits)
Enterprised ID(32 bits)

Reply:
Due to space concerns, for first bit I didn't add the `(1bit)` to the diagram, but under it. And changed the element length for field length and enterprise id (previously I didn't copy the diagram correctly). Thanks!